meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Upstream PostgreSQL client
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.6
+  min_ansible_version: '2.6'
```

Comment:
2.10 as discussed ?
similarly to
https://github.com/ome/ansible-role-fluentd/blob/d1a5c0e19bdccdb0c8ae17f44c8f136bb449477e/meta/main.yml#L7 - according